src/services/completions.ts

```diff
@@ -10,7 +12,17 @@ namespace ts.Completions {
         AutoImportSuggestions = "6",
         JavascriptIdentifiers = "7"
     }
-    export type Log = (message: string) => void;
+
+    enum SortTextId {
+        LocalDeclarationPriority,
+        LocationPriority,
+        OptionalMember,
+        MemberDeclaredBySpreadAssignment,
+        SuggestedClassMembers,
+        GlobalsOrKeywords,
+        AutoImportSuggestions,
+        Deprecated
+    }
 

```

Comment:
```ts
    const DeprecatedSortTextStart = SortTextId.AutoImportSuggestion; + 2; // for Javascript identifiers since with this change they are preferred over deprecated symbols
```